textattack/shared/attacked_text.py

```diff
@@ -376,6 +376,11 @@ def text(self):
         """
         return "\n".join(self._text_input.values())
 
+    @property
+    def num_words(self):
+        """ Returns the number of words in the sequence. """
+        return len(self.words)
+
```

Comment:
I think this is worth being its own method - but there's probably a lot of uses of `len(at.words)` still scattered about the codebase, right?

if so, can you put a task on asana and/or github issue to clean up the code and replace them all with `at.num_words`?